tests/assign/moverload_asgn2.nim

```diff
@@ -1,3 +1,7 @@
+discard """
```

Comment:
FYI discard sections for files not starting with 't' are not processed in any way.